flaml/automl.py

```diff
@@ -599,6 +604,13 @@ def _prepare_data(self,
                         y_train_all,
                         test_size=split_ratio,
                         random_state=RANDOM_SEED)
+            elif eval_method == 'holdout' and self._state.task == 'forecast':

```

Comment:
for classification & regression holdout, when split_type='time', need to do it in a similar way as forecasting.